src/state/reducer.js

```diff
@@ -10,16 +11,16 @@ import revisions from 'console/state/revisions/reducers';
 import serviceInfo from 'console/state/serviceInfo/reducers';
 import users from 'console/state/users/reducers';
 
-const reducer = combineReducers({
-  actions,
-  approvalRequests,
-  auth,
-  extensions,
-  recipes,
-  network,
-  revisions,
-  serviceInfo,
-  users,
-});
-
-export default reducer;
+export default history =>
```

Comment:
Why call it "history"? ...and not "reducer"? Isn't the outcome of `combineReducers` another reducer? I know that JavaScript's `export default` voids the importance of a consistent name, but still. It's harder to debug this when the names don't add up I think.

Reply:
this is a param of the arrow function. it is the history that is being passed through.

Reply:
Oh silly me. I read that too quickly.